packages/p2p/src/components/my-ads/create-ad-add-payment-method-modal.jsx

```diff
@@ -81,16 +76,24 @@ const CreateAdAddPaymentMethodModal = () => {
                     {localize('Add payment method')}
                 </React.Fragment>
             }
-            toggleModal={onCancel}
+            toggleModal={e => {
+                if (
+                    !e.target ||
```

Comment:
There is an issue with the `Modal` component when `Autocomplete` dropdowns are integrated within its body: it detects anything clicking the dropdown item as clicking outside the modal, thus calling `toggleModal`.